internal/pkg/cli/env_deploy.go

```diff
@@ -215,5 +217,6 @@ Deploy an environment named "test".
 	}
 	cmd.Flags().StringVarP(&vars.appName, appFlag, appFlagShort, tryReadingAppName(), appFlagDescription)
 	cmd.Flags().StringVarP(&vars.name, nameFlag, nameFlagShort, "", envFlagDescription)
+	cmd.Flags().BoolVar(&vars.forceNewUpdate, forceFlag, false, forceFlagDescription)
```

Comment:
I think we need to update the `forceFlagDescription` right now it's specific to services. Something like:
```go
const forceEnvDeployFlagDescription = "Optional. Force update the environment stack template." 
```